### PR TITLE
Add error message to rsync.sh if read_files.sh fails

### PR DESCRIPTION
**Pull request by mimi1vx**
https://progress.opensuse.org/issues/112871

### CODE REVIEW

File: script/rsync.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ set +e
     [ -e "$subfolder/print_openqa.sh" ] || { >&2 echo "No file found: {$subfolder/print_openqa.sh}"; exit 1; }
 
     # nowhere to log yet as we haven't created $logfolder
-    bash -e "$subfolder/read_files.sh"
+    bash -e "$subfolder/read_files.sh" ||  { >&2 echo "read_files.sh failed for $ubfolder in enviroment $environ"; exit 1; }
```
Comment (kalikiana) [inline script/rsync.sh:39]:
```suggestion
    bash -e "$subfolder/read_files.sh" ||  { >&2 echo "read_files.sh failed for $subfolder in enviroment $environ"; exit 1; }
```

Where is `$environ` defined? I don't see it in this repo 🤔

Comment (perlpunk) [reply script/rsync.sh:39]: in line 4 of this script

Comment (kalikiana) [reply script/rsync.sh:39]: Oh, thanks. Of course `grep $environ` failed me 🤦🏾‍♀️

Comment (mimi1vx) [reply script/rsync.sh]: thx, typo :D

**Review by perlpunk — APPROVED** [review]

**Pull request merged**